src/tonie_api/api.py

```diff
@@ -29,13 +29,20 @@ class TonieAPI:
 
     API_URL = "https://api.tonie.cloud/v2"
 
-    def __init__(self, username: str, password: str) -> None:
+    def __init__(self, username: str, password: str, user_agent: str) -> None:
         """Initializes the API and creates a session token for tonie cloud session."""
+        if not user_agent or user_agent == "":
+            self.user_agent = "tonieApi/2.0"
+        else:
+            self.user_agent = user_agent

```

Comment:
I haven't found anything in the api regarding the `User-Agent` Header. But I also don't see anything which should block it.

```python
 def __init__(self, username: str, password: str, user_agent: str = "tonieApi/2.0") -> None:
        """Initializes the API and creates a session token for tonie cloud session."""
        self.user_agent = user_agent
```

Should also fix the tests.

Reply:
Hi @Wilhelmsson177 Willi

I prepare an update and tests but please allow me some time to do.

Tom